src/components/RetroItem.vue

```diff
@@ -79,16 +80,24 @@
               </div>
             </template>
           </draggable>
-
         </div>
-
-
       </div>
+
       <button @click="submit" class="btn-primary bg-next animate__pulse">
         <p class="txt-neu fz-h2">Submit</p>
       </button>
     </div>
 
+    <AlertTheme v-show="isShow" @closeAlert="isShow = false" :alertMessage="alertMessage"
+                :alert-btn-message="alertBtnMessage">
+      <template></template>
+    </AlertTheme>
+
+    <ConfirmTheme v-show="isShow" @closeConfirm="isShow = false" :cancel-btn-message="cancelBtnMessage"
```

Comment:
alert and confirm both using isShow attribute to control show or not, then if isShow change to true, both these two component will show